src/app/invest/invest.component.scss

```diff
@@ -102,21 +82,12 @@ p {
 }
 
 .estimated {
-
-  &__table {
-    display: flex;
-    flex-direction: column;
-  }
-
```

Comment:
I guess this was unused because there was not estimated__table element in HTML, this CSS is removed